policyengine_us/parameters/gov/states/mo/tax/income/credits/wftc/match.yaml

```
@@ -1,10 +1,13 @@
 description: Missouri's Working Families Tax Credit matches this percent of the federal EITC.
```

Avoiding acronyms in descriptions while here
```suggestion
description: Missouri's Working Families Tax Credit matches this percent of the federal Earned Income Tax Credit.
```